transformer/src/getSignatureParameters.ts

```diff
@@ -35,7 +35,9 @@ export function getSignatureParameters(signature: ts.Signature, context: Context
 		parameters.push({
 			n: parameterSymbol.getName(),
 			t: type && getTypeCall(type, type.symbol, context, getCtorTypeReference(parameterSymbol)) || getUnknownTypeCall(context),
-			o: declaration.questionToken !== undefined || declaration.initializer !== undefined,
+			o: declaration.questionToken !== undefined || declaration.initializer !== undefined || declaration.dotDotDotToken !== undefined,
+			var: !!declaration?.dotDotDotToken,
+			dv: type?.isClass() ? null /*TODO: handle class, but how ? */ : declaration.initializer,
```

Comment:
@Hookyns, do you know how to handle `private foo: Foo = new Foo()` here?

I can't find a documentation link for this part of typescript...

Reply:
What do you mean "handle it"? detecting runtime values of parameters is out of scope I think.